Libraries/LibWeb/HTML/Dates.cpp

```diff
@@ -90,10 +90,17 @@ bool is_valid_month_string(Utf16View const& value)
     if (parts[1].length_in_code_units() != 2)
         return false;
 
-    for (auto digit : parts[0])
+    auto year_is_zero = true;
+    for (auto digit : parts[0]) {
         if (!is_ascii_digit(digit))
             return false;
 
+        if (digit != '0')
+            year_is_zero = false;
+    }
+    if (year_is_zero)
+        return false;
```

Comment:
I think this code would be a bit more readable if it just said:

```cpp
if (parts[0] == "0000"sv)
    return false;
```

Reply:
Ah woops, didn't realize I edited `is_valid_week_string` there - but yeah, I think it's clearer to just do it this way in both methods, since they both need to check that the year is non-zero.